gemsfx/src/main/java/com/dlsc/gemsfx/util/SessionManager.java

```diff
@@ -63,13 +68,15 @@ public void register(String path, DoubleProperty property) {
     public void register(String path, IntegerProperty property) {
         LOG.fine("registering integer property at path " + path);
         property.set(preferences.getInt(path, property.get()));
-        property.addListener((it, oldValue, newValue) -> {
+        initializePreferences(path, property);
+        var listener = (ChangeListener<Number>) (it, oldValue, newValue) -> {
```

Comment:
We do not use the var keyword in GemsFX.

Reply:
Fixed. May  I ask why?

Reply:
Convention ..... as in "I do not use it". I prefer reading declarations left to right. Just what I am used to.